packages/lutin/lutin.2.56/opam

```diff
@@ -25,8 +25,10 @@ remove: [
   ["ocaml" "%{etc}%/Lutin/setup.ml" "-C" "%{etc}%/Lutin" "-uninstall"]
 ]
 depends: [
-  "ocaml" {>= "4.02" & < "4.12"}
+  "ocaml" {>= "4.04" & < "4.06" }
```

Comment:
these new constraints seem highly suspisious. Failures on those specific already established packages can be ignored for now. Could you move this to another PR? (same for 2.70.4)